demos/demo_sftp.py

```diff
@@ -79,7 +79,7 @@
     try:
         # try ~/ssh/ too, because windows can't have a folder named ~/.ssh/
         host_keys = paramiko.util.load_host_keys(
-            os.path.expanduser("~/ssh/known_hosts")
+            os.path.expanduser("~/.ssh/known_hosts")
```

Comment:
this path is attempted above on line 76, the explanation for trying again without this dot is on line 80